gatsby-node.js

```diff
@@ -1,17 +1,54 @@
-const path = require("path");
+const path = require('path');
+const remark = require('remark');
+const remarkHTML = require('remark-html');
+
+const frontmatterFieldSettings = [
+  { field: 'description', defaultValue: null, supportMarkdown: true },
+  { field: 'location', defaultValue: null },
+  { field: 'people', defaultValue: null, supportMarkdown: true },
+  { field: 'recording', defaultValue: null, supportMarkdown: true },
+  { field: 'time', defaultValue: null },
+];
+
+exports.onCreateNode = ({ node }) => {
+  if (node.frontmatter) {
+    frontmatterFieldSettings.forEach(
+      ({ field, defaultValue, supportMarkdown }) => {
+        const value = node.frontmatter[field];
+
+        if (supportMarkdown && value) {
+          node.frontmatter[field] = remark()
```

Comment:
Interesting, so this is to support markdown in text values of frontmatter? But stripping paragraph tags?

Reply:
Yep! Kinda sucks to need to strip the paragraph tags, but it's needed cause I want to use this directly inside an `<li>` without messing up the formatting...